CAAppKitBridge/Source/GNUmakefile

```diff
@@ -0,0 +1,36 @@
+
+ifeq ($(GNUSTEP_MAKEFILES),)
+  GNUSTEP_MAKEFILES := $(shell gnustep-config --variable=GNUSTEP_MAKEFILES 2>/dev/null)
+  ifeq ($(GNUSTEP_MAKEFILES),)
+    $(error You need to set GNUSTEP_MAKEFILES before compiling!)
+  endif
+endif
+
+include $(GNUSTEP_MAKEFILES)/common.make
+#
+# Main framework
+#
+FRAMEWORK_NAME =CAAppKitBridge
+
+# Include configuration
+
+-include ../config.make
+
+CAAppKitBridge_OBJCFLAGS += -std=c99
+#CAAppKitBridge_HEADER_FILES_DIR = ../Headers/CAAppKitBridge
```

Comment:
This project will almost certainly have no public headers. You can remove this for now.